app/models/order.rb

```diff
@@ -0,0 +1,19 @@
+class Order < ApplicationRecord
+
+  belongs_to :restaurant
+  belongs_to :customer
+  has_many :line_items, dependent: :destroy, after_add: :increase_total, after_remove: :decrease_total
```

Comment:
what if update

Reply:
Fixed.
Method called using after_save callback in line_item.